src/hooks/protections/received_clone_create.cpp

```diff
@@ -12,6 +13,13 @@ namespace big
 			return;
 		}
 
+		if (*g_pointers->m_gta.m_clone_create_pool && (*g_pointers->m_gta.m_clone_create_pool)->m_size < 2)
+		{
+			// We don't have enough memory to handle this
+			g_notification_service.push_warning("Protections", "Low net object pool size");
```

Comment:
can we have similar check in `received_clone_sync` just curious?

Reply:
Not necessary as blocking it in create solves the problem already.